index.html

```diff
@@ -8,7 +8,11 @@
 <body>
     <canvas id="canvas"></canvas>
     <img src="static/images/player.png" id="playerSprite" style="display: none;">
+    <img src="static/images/player_mirror.png" id="playerSpriteMirrored" style="display: none;">
```

Comment:
i wonder if we should be mirroring using context transforms instead of separate images